src/Isotope/Model/Payment/Wirecard.php

```diff
@@ -99,7 +99,7 @@ public function checkoutForm(IsotopeProductCollection $order, Module $module)
             'request-id' => $order->getUniqueId(),
             'transaction-type' => self::$paymentMethods[$this->wirecardPaymentMethod],
             'requested-amount' => [
-                'value' => $order->getTotal(),
+                'value' => number_format($order->getTotal(), 2),
```

Comment:
```suggestion
                'value' => number_format($order->getTotal(), (int) \Isotope\Isotope::getConfig()->priceRoundPrecision ?: 2, '.', ''),
```
I have also changed the thousands separator, otherwise it will lead to invalid numbers. Could you please test this change with cart total that is equal or greater than 1000? e.g. something like `1000.1`.

Reply:
Thanks!

Reply:
Just to be clear: I haven't made any actual changes yet 😁 Can you test the proposed change the way I described?